staff/pablo-rey/spotify-template/style-mobile.css

```diff
@@ -0,0 +1,117 @@
+@import url("https://fonts.googleapis.com/css?family=Montserrat:400,600,800");
+* {
+  box-sizing: border-box;
+}
+
+html, body {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  font-family: Montserrat;
+  font-size: 14px;
```

Comment:
@paulusrex as a recommendation, when working with responsive sites, `em` or `rem` should be used. Pixels will cause you a headache!

Reply:
With this font-size i am defining the rem value. correct?

Reply:
With this font-value you are setting the font-size of all document to 14px. If you do not specify it anywhere else, it will always be 14px (mobile, tablet,..). Once you set font-size in the html, like in here, if you need the font-size to change throughout the site, you can use rem or em:
em - is relative to the font size of its direct or nearest parent.
rem - is only relative to the html (root) font-size.
If you do not set it in the html, by default is 16px (=1rem). Then, you can set h1 to 1.2rem,... mobile: 0.9rem, desktop: 1.1rem,...